### ISSUE TITLE
server: should warn when running with known kernel bugs

### ISSUE THREAD

**Issue by tbg** [cockroachdb/cockroach#32906]
We encountered an issue today (stuck timers) that was traced down to a kernel bug:

Problem appears on the 4.13.0-45 kernel
lore.kernel.org/lkml/tip-1f71addd34f4c442bec7d7c749acc1beb58126f2@git.kernel.org is the fix

There was also another old kernel that caused networking issues (@nvanbenschoten do you remember/can you retrieve the patch fixing that one? Something about "riding the rocket" in dmesg).

CRDB should warn (to the logs, regularly unless disabled by cluster setting/env var) if it suspects that it's running on a kernel susceptible to any bugs we know have impacted proper functioning of CRDB.

See also https://github.com/cockroachdb/cockroach/issues/32905

**Comment by nvanbenschoten** [comment]
Yeah, this is related to https://github.com/cockroachdb/cockroach/issues/20311. We can close one of them in favor of the other.

**Comment by knz** [comment]
we should log the output of `uname` in any case
